client/package.json

```diff
@@ -119,7 +119,7 @@
 		"path-browserify": "1.0.0",
 		"percentage-regex": "3.0.0",
 		"phone": "2.4.2",
-		"photon": "file:../packages/photon",
+		"photon": "^3.0.0",
```

Comment:
We probably want to use `^` syntax everywhere there was a `file:` reference - otherwise everytime a local package is bumped we'll have to go and update all the references to it.

Reply:
🤔 what happens if local version is bumped v4 in this case? Would we then want to use `*` instead?

Reply:
Yeah for private packages like `client/*` it might be a good idea to encourage the developer who makes a change to also update any usages across the codebase.

In packages published to the registry, `*` would have unintended affects.

Reply:
I'm all for adding a version constraint that enforces workspaces are always symlinked (the goal of `^` was to keep it symlinked for longer). I tried `yarn@2` with its constraints in mind but the ecosystem isn't stable enough yet. Maybe we can use `manypkg` to enforce that for now?